lethal-squirrels/workforce-management
# made first name and start date non editable

## REFERENCE

Ticket: #36

## DESCRIPTION
Made first name and start date non editable on the Edit Employee page

## STEPS TO TEST
1. Clone ajfirstname to a local repo
2.Test functionality
	* Go to the Edit Employee page and make sure data shows up correctly on the page and that 
           first name and start date are non editable. Make sure you can save changes in other fields too
